ladok_generate_certificate_verify_user.sh

```diff
@@ -73,8 +73,13 @@ fi
 
 if ! choose_ladok_environment "${certificate_name}" "${password_input}"; then
     printf "\tERROR no ladok environment was found\n"
+    if openssl x509 -in "${certificate_name}".crt -passin pass:"${password_input}" -subject -noout | grep Prod-UFH; then
+        printf "\t%s\n" "Uppföljningsdatabasen cert, no REST available, use STUNNEL." \
```

Comment:
printf "\t%s\n" -> printf "\t%s\n\t%s\n\t%s\n" since there is three arguments.

Reply:
Not necessary, this works fine =)